pkgs/servers/mail/mailman/web.nix

```diff
@@ -1,37 +1,38 @@
-{ stdenv, python, hyperkitty, postorius, buildPythonPackage
-, serverEMail ? "postmaster@example.org"
-, archiverKey ? "SecretArchiverAPIKey"
-, allowedHosts ? []
+{ buildPythonPackage, lib, fetchgit
+, git, makeWrapper, sassc, hyperkitty, postorius
 }:
 
-let
-
-  allowedHostsString = stdenv.lib.concatMapStringsSep ", " (x: "\""+x+"\"") allowedHosts;
-
-in
-
-# We turn those Djando configuration files into a make-shift Python library so
-# that Nix users can use this package as a part of their buildInputs to import
-# the code. Also, this package implicitly provides an environment in which the
-# Django app can be run.
-
-buildPythonPackage {
-  name = "mailman-web-0";
+buildPythonPackage rec {
+  pname = "mailman-web-unstable";
+  version = "2019-09-29";
+
+  src = fetchgit {
```

Comment:
not sure if you already explored this, but there is a fetchFromGitLab helper. Generally they are less expensive because you just get an archive instead of an entire git history

Reply:
Doesn’t do `leaveDotGit`.